packages/cli/tests/cli/add.test.ts

```diff
@@ -19,6 +19,8 @@ describe('Add command', () => {
         '0x2E645469f354BB4F5c8a05B3b30A929361cf77eC',
         '--contract-name',
         'Gravatar',
+        '--start-block',
```

Comment:
Add a separate test for case when startBlock is not supplied to avoid NaN bug above